### ISSUE TITLE
Design: Reduce community head stat box size

### ISSUE THREAD

**Issue by sevonj** [Xyphyn/photon#447]
**Is your feature request related to a problem? Please describe.**
So, 360px width is just a tiny bit too little to make the stats fit in a 2x2 grid. In 368px they do.

**Describe the solution you'd like**
Reduce margin or size or something to make 2 stat blocks fit horizontally on a 360px wide screen. 

**Describe alternatives you've considered**
\-

**Additional context**
|368px|360px|
|--|--|
|![image](https://github.com/user-attachments/assets/7a86218b-6b1b-4ba4-8b59-73aa35d1796f)|![image](https://github.com/user-attachments/assets/532df498-ba57-4a24-aa06-f8de5b2078a7)|

**Comment by jansteffen** [comment]
I think that this whole box is honestly unnescessary and even on desktop it takes up too much space:
![image](https://github.com/user-attachments/assets/dc029056-602d-4c2a-abaf-910599ae5078)

That's literally half the page height, even on a 1440p monitor. Just to present some info that's *already* being presented in the sidebar. Sure, it looks fancy with the blur effect, but in terms of navigation and useability I honestly think it's a regression.

**Comment by sevonj** [comment]
> but in terms of navigation and useability I honestly think it's a regression.

I disagree, I think the the header is a better place to show the info than the sidebar, which is inaccessible on narrow screens.

Some vertical compacting would on desktop too would help though.

**Comment by Xyphyn** [comment]
This info has to be accessible somewhere but I don't know where I'd put it on mobile

**Comment by jansteffen** [comment]
My suggestion would be to make in an expandable that is collapsed by default. When collapsed it only shows the banner image, community logo and name.

**Comment by BrikoX** [comment]
Adding community description below the name takes up valuable space for no reason and Markdown parsing from links to lists being rendered breaks the visual flow. 

Full banner size, with description takes up more than half the page on 1080p desktop. It literally leaves enough space to see a single post. 
![image](https://github.com/user-attachments/assets/b009c112-ef4b-4a24-bbf5-f6128c080641)

Reduced banner size, with no description still takes up 1/3 of the page on 1080p desktop. At least you can see 3 posts now.
![image](https://github.com/user-attachments/assets/5772f585-d169-44ff-8b51-e0f586e419e5)

---

> This info has to be accessible somewhere but I don't know where I'd put it on mobile

@Xyphyn since iOS finally added proper gesture support on PWA, you make the sidebar accessible that way. It's also already what most mobile user are used to.

**Comment by Xyphyn** [comment]
The gestures conflict with Android though. I'll work on the header later today, im really stressed with hwk.

**Comment by sevonj** [comment]
![image](https://github.com/user-attachments/assets/c1e0dd12-9f92-4a75-9e93-bed93aea6840)
I think closing this gap by reducing height would definitely help. It should be an easy change, except what is with mask-image affecting the other content too?

![image](https://github.com/user-attachments/assets/ccfbbc2e-5a1a-4583-bf7e-782dcacd45b2)
Maybe subscribe and actions buttons could be somehow relocated to the free space in the banner which would eliminate one row. This seems like annoying to get right on all devices though, it would have to wrap below on portrait mobile.

**Comment by Xyphyn** [comment]
Your cropped screenshot just gave me an idea for a more compact design hold up

**Comment by Xyphyn** [comment]
The design was altered and uses less space.
